File: Java/Disk.java
public class Disk {
    //varibel untuk menampung tipe, kapasitas, dan harga
    private String type;       
    private String capacity;    
    private int price;          

    // constructor
    public Disk() {
    }

    public Disk(String type, String capacity, int price) {
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }

    // setter and getter
    public void setType(String type) {
        this.type = type;
    }

    public String getType() {
        return this.type;
    }

    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }

    public String getCapacity() {
        return this.capacity;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return this.price;
    }

    // method
    public void printDisk() {
        System.out.println("Tipe Disk : " + this.getType());
        System.out.println("Kapasitas Disk : " + this.getCapacity());
        System.out.println("Harga Disk : Rp " + this.getPrice());
    }
}

File: Python/Disk.py
class Disk:
    #varibel untuk menampung tipe, kapasitas, dan harga
    __type = ""     
    __capacity = "" 
    __price = 0     

    # constructor
    def __init__(self, type = "", capacity = "", price = 0):
        self.__type = type
        self.__capacity = capacity
        self.__price = price

    # setter and getter
    def setType(self, type):
        self.__type = type
    
    def getType(self):
        return self.__type

    def setCapacity(self, capacity):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity
    
    def setPrice(self, price):
        self.__price = price
    
    def getPrice(self):
        return self.__price

    # method
    def printDisk(self):
        print("Tipe Disk : " + str(self.getType()))
        print("Kapasitas Disk : " + str(self.getCapacity()))
        print("Harga Disk : Rp " + str(self.getPrice()))